epigraphhub/data/upload.py

```diff
@@ -9,12 +9,19 @@ def upload_geo_file(
 ) -> None:
```

Comment:
Maybe this module could be renamed to something more meaningful. It seems to be handling just geo data, so maybe its name could be `geoupload.py`?

Reply:
Ok